spec/forms/steps/caution/conditional_end_date_form_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
 
 RSpec.describe Steps::Caution::ConditionalEndDateForm do
-  it_behaves_like 'a date question form', attribute_name: :conditional_end_date
+  it_behaves_like 'a date question form', attribute_name: :conditional_end_date, future_date: true
```

Comment:
Can we name this to `allow_future`, so it is consistent with the intended validation.